addons/dango-rain/addon.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Dango rain on profiles",
+  "description": "Profiles that include \"dango\" on their \"About Me\" section will have dangos raining on the sides of the screen!",
+  "info": [
+    {
+      "type": "notice",
+      "text": "If you disable this addon, you will only be able to find it if you enter the konami code.",
```

Comment:
I don't think we want to tell them how to get the easter eggs addons...

Reply:
1. this is on the todo
2. we actually want users to know about the addons... why wouldn't we...